jetty-core/jetty-deploy/src/main/config/modules/deploy.mod

```
@@ -1,15 +1,10 @@
 [description]
-This module enables web application deployment from the `$JETTY_BASE/webapps` directory.
+Deprecated `deploy` module (does nothing on its own)
```

Perhaps better to remove it.
Would give a fail fast error, rather than Jetty starting fine, but then not deploying anything.

---

We have the deprecation features for this reason.
We don't remove the module in other similar cases.

---

We deprecated, but kept the functionality.
This is deprecation, with modified behavior. At this point better to remove.

---

We deprecated other modules, but kept behavior.
In this case, the behavior is different, so I think removing the module is better.